cmd/skywire-cli/commands/config/gen.go

```diff
@@ -261,6 +335,32 @@ var genConfigCmd = &cobra.Command{
 		}
 		//don't write file with stdout
 		if !stdout {
+			thisUser, err := user.Current()
+			if err != nil {
+				panic(err)
```

Comment:
no need to really error here & Ive changed it to fail transparently. This is only checking for root and it will just not change the default from false to true.